src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp

```diff
@@ -66,10 +66,8 @@ class ConvLowPrecisionTest : public testing::WithParamInterface<ConvLowPrecision
     }
 
     InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo& info) const override {
-        return FuncTestUtils::createAndFillBlob(info.getTensorDesc(),
-                                                -0.05f,
-                                                0.05f,
-                                                1 / inputDataResolution);
+        return FuncTestUtils::createAndFillBlobFloatNormalDistribution(
+            info.getTensorDesc(), 0.0f, 0.2f, 7235346);
```

Comment:
`7235346` looks like a magic number.
wouldn't e.g. `7e6` be fine as well? or maybe even 1e4 would be enough

Reply:
Yes, both your numbers are perfect (not tested), however `7235346` looks even better, because it was used in all other calls to `createAndFillBlobFloatNormalDistribution` existing in the project. So what's the reason to break this convention? If you wish, however, I can add field (say `seed`) to store this value :)